src/app/components/new/new.component.ts

```diff
@@ -22,7 +30,35 @@ export class NewComponent {
     }
     return false;
   }
+
+  checkIfCommandDone(commandId: string): void {
+    this.commandService.isCommandDone(commandId)
+      .subscribe(response => {
+        this.isCommandDone = !!response;
```

Comment:
pseudo-code:

```typescript
if (!response) {
  <time out for 1000ms> (timeout / rxjs alternative)
  this.checkIfCommandDone(commandId);
} else {
  this.changeAngularProjectStatus.emit();
}
```
don't forget error handling!

Reply:
If we do this we no longer need the interval.. or am I missing something?

Reply:
@ortichon